lib/Support/Debug.cpp

```diff
@@ -89,6 +89,7 @@ namespace llvm {
           memcpy(chunk, Ptr, len);
           chunk[len] = '\0';
           OutputDebugStringA(chunk);
+          printf(chunk);
```

Comment:
>printf(chunk); [](start = 10, length = 14)

For local debugging?

Reply:
Yes. I'll get rid of it before I check it in.